### ISSUE TITLE
Remove button for compliance profiles in Automate UI

### ISSUE THREAD

**Issue by Rajankumarprgrs** [chef/automate#5295]
Remove button for compliance profiles in Automate UI.

Steps to replicate:-

1.Login to automate UI.
2.Go to Compliance page.
3.Click on profiles.
4.Add any compliance profile.
5.Now it will list under profiles tab.
6.Try to remove the added profile.

Expected Result:-remove button should be there itself on the same page.

Actual result:-Currently we need to go one level down i.e. we need to click on the profile link then we will get that remove button.
Refere the screenshot:-

<img width="1683" alt="Screenshot 2021-06-28 at 4 05 03 PM" src="https://user-images.githubusercontent.com/73093307/123630337-085a4f00-d833-11eb-90d7-e66914fd5a45.png">

**Comment by AnkurMundhra** [comment]
@jong-progress What is your take on this, Jon?

**Comment by jong-progress** [comment]
> @jong-progress What is your take on this, Jon?

@AnkurMundhra 

Yeah, we designed this ages ago, but guess it never was implemented. To take it even further, we should follow the UI pattern that Infrastructure uses, when removing Nodes. Where a user can also multi-select profiles and do a group delete.

I dug out an old Mock Up.
![Screen Shot 2021-06-28 at 12 47 41 PM](https://user-images.githubusercontent.com/79271174/123695328-4776a900-d80f-11eb-952f-d7ab6e303894.png)
